chemicaltoolbox/datatypes/molecules.py

```diff
@@ -449,6 +449,36 @@ def set_peek( self, dataset, is_multi_byte=False ):
             dataset.peek = 'file does not exist'
             dataset.blurb = 'file purged from disk'
 
+class PDBQT(GenericMolFile):
+    file_ext = "pdbqt"
+

```

Comment:
Can you please add a link or short description as comment to the file type? This will serve as documentation. PDBqt it probably well known, but LOG I never heard about it :)